docs/sql-keywords.md

```diff
@@ -159,6 +160,7 @@ Below is a list of all the keywords in Spark SQL.
   <tr><td>LIMIT</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
   <tr><td>LINES</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
   <tr><td>LIST</td><td>non-reserved</td><td>non-reserved</td><td>non-reserved</td></tr>
+  <tr><td>LIST_VALUE</td><td>non-reserved</td><td>non-reserved</td><td>reserved</td></tr>
```

Comment:
This is wrong because you want `LAST_VALUE`. :)

Reply:
I think the two words should be reserved in spark (ansi=true). Also, you need to update `TableIdentifierParserSuite`.